applications/serial_lte_modem/doc/slm_description.rst

```diff
@@ -435,6 +441,28 @@ To connect with an external MCU using UART_2, change the configuration files for
 
 .. tabs::
 
+   .. group-tab:: nRF9151 DK
+
+      * In the :file:`nrf9151dk_nrf9151_ns.conf` file::
+
+          # Use UART_0 (when working with PC terminal)
+          # unmask the following config
+          #CONFIG_UART_0_NRF_HW_ASYNC_TIMER=2
+          #CONFIG_UART_0_NRF_HW_ASYNC=y
+          #CONFIG_SLM_POWER_PIN=8
+          #CONFIG_SLM_INDICATE_PIN=0
+
+          # Use UART_2 (when working with external MCU)
+          # unmask the following config
+          CONFIG_UART_2_NRF_HW_ASYNC_TIMER=2
+          CONFIG_UART_2_NRF_HW_ASYNC=y
+          CONFIG_SLM_POWER_PIN=31
+          CONFIG_SLM_INDICATE_PIN=30
+
+      * In the :file:`nrf9151dk_nrf9151_ns.overlay` file::
+
+          #include "nrf9161dk_nrf9161_ns.overlay"
```

Comment:
This part should reflect the changes required for the overlay file to use the UART2. This is a bit tricky to describe in here as we include the 9161 overlay file. What the user should do at this point would be either to do the changes to 9161 file or copy the whole 9161 overlay file in here and make the similar changes.

I think that I will create a PR to separate the 9151 overlay file from the 9161. It is a bit of duplication but easier to use for the end-user, so ** this part would have the same information than the 9161.**

Also, ping @tomi-font